test/e2e_node/cpu_manager_test.go

```diff
@@ -179,12 +167,6 @@ func getCPUSiblingList(cpuRes int64) string {
 	return string(out)
 }
 
-func getCoreSiblingList(cpuRes int64) string {
-	out, err := exec.Command("/bin/sh", "-c", fmt.Sprintf("cat /sys/devices/system/cpu/cpu%d/topology/core_siblings_list | tr -d \"\n\r\"", cpuRes)).Output()
-	framework.ExpectNoError(err)
-	return string(out)
-}
-
```

Comment:
this should be restored and reused in the future. Ideally, we need to check:
1. we get the number of CPUs we asked; (done already)
2. these CPUs are thread siblings (must-have for full-pcpus-only option; nice-to-have otherwise)
3. the container cgroup contains ONLY the CPUs we asked (we need to navigate the cgroup layout)